global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
     "rollForward": "feature",
-    "version": "6.0.100"
+    "version": "6.0.303"
```

Comment:
Latest sdk is `6.0.400`. Why 303?

Reply:
great point :) I ran `dotnet --list-sdks` on my machine and it was the newest non-prerelease 😅